news/2023-01-04-featured-artist-track-updates-winter-2023.md

```diff
@@ -0,0 +1,378 @@
+---
+layout: post
+title: "Featured Artist Track Updates: Winter 2023"
+date: 2023-01-04 13:00:00 +0000
+---
+
+Your New Year's resolution is to learn how to map, right? We've picked up over **60** new songs from **18** of our Featured Artists for you to try out!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+We announce new Featured Artists twice a week. Artists never stop producing music though, so we sometimes pick up songs from our backlog of artists. This post is a compilation of every new song we've licensed since [our batch last Summer](https://osu.ppy.sh/home/news/2022-07-02-featured-artist-track-updates-summer-2022).
```

Comment:
```suggestion
We announce new Featured Artists twice a week. Artists never stop producing music though, so we sometimes pick up songs from our backlog of artists. This post is a compilation of every new song we've licensed since [our batch last summer](https://osu.ppy.sh/home/news/2022-07-02-featured-artist-track-updates-summer-2022).
```